examples/api/lib/widgets/sliver/decorated_sliver.1.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+// Flutter code example for [DecoratedSliver]
+// with clipping turned off in a parent [CustomScrollView].
```

Comment:
Wrapping a class name with [square brackets] creates a link to that class, both in the IDE and on the [docs website](https://main-api.flutter.dev/), but it needs to be structured as a doc comment for this to work.

```suggestion
/// Flutter code example for [DecoratedSliver]
/// with clipping turned off in a parent [CustomScrollView].
```